package.json

```diff
@@ -9,7 +9,7 @@
     "grunt-contrib-concat": "~0.4.0",
     "grunt-contrib-copy": "~0.5.0",
     "grunt-contrib-cssmin": "~0.9.0",
-    "grunt-contrib-jshint": "~0.6.0",
+    "grunt-eslint": "^7.0.1",
```

Comment:
Ah, didn't realize Node had gone to caret. Ironically, Node itself introduces backward breaks in minor releases, as do many Node projects (looking at you Grunt)